system_suitesparse/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<package>
+  <name>system_suitesparse</name>
+  <version>4.2.1</version> 
+  <description>CMake scripts finding system suitesparse library</description>
+  <maintainer email="hannes.sommer@mavt.ethz.ch">Hannes Sommer</maintainer>
+  <license>LGPLv3 or later</license>
+
+  <author>Dr. Tim Davis</author>
```

Comment:
An author is required?

Reply:
If not we could simply remove it (doesn't make that much sense for such an indirecting package).